packit/config/package_config.py

```diff
@@ -273,8 +290,37 @@ def parse_loaded_config(
             raw_dict=loaded_config,
             config_file_path=config_file_path,
             repo_name=repo_name,
+            spec_file_path=spec_file_path,
         )
         return package_config
     except Exception as ex:
         logger.error(f"Cannot parse package config. {ex}.")
         raise PackitConfigException(f"Cannot parse package config: {ex}.")
+
+
+def get_local_specfile_path(directories: Union[List[str], List[Path]]):
+    """
+    Get the relative path of the local spec file if present.
+    :param directories: dirs to find the spec file
+    :return: str relative path of the spec file
+    """
+    for dir in directories:
+        files = [
+            os.path.relpath(path, dir) for path in glob(os.path.join(dir, "*.spec"))
+        ]
+        if len(files) > 0:
+            return files[0]
+
+    return None
+
+
+def get_specfile_path_from_repo(project: GitProject):
+    """
+    Get the path of the spec file in the given repo if present.
+    :param project: GitProject
+    :return: str path of the spec file
+    """
+    try:
+        return project.get_files(filter_regex=".*.spec")[0]
+    except Exception:
```

Comment:
please don't eat the exception, they don't taste well - print it to debug log at least